Dockerfile

```diff
@@ -37,8 +37,18 @@ RUN git config --global credential.credentialStore cache
 
 # Allow development-env to be updated from within the container
 ARG DEVELOPMENTENVDIR="/development-env"
+# TODO - it's fine to have the development environment all available for editing
+# but we should also have a separate folder for the things that we need to call
+# regularly - separate from the stuff which we don't. Or aliases / symlinks to them.
```

Comment:
Yes, but this will be addressed in a subsequent merge request. Remove this comment.